lib/Button/Button.less

```diff
@@ -16,6 +27,14 @@
 	padding: 0 @moon-button-h-padding;
 	margin: 0 @moon-spotlight-outset;
 	color: @moon-button-text-color;
+	-webkit-transform: translateZ(0);
+	-webkit-transition: transform 0.2s ease-out;
+	transform: translateZ(0);
+	transition: transform 0.2s ease-out;
+	-webkit-animation-duration: 0.2s;
```

Comment:
If we did, it must have been to support future browsers. Chrome/blink/webkit doesn't yet. Firefox may, now that I think about it... We should check.